### ISSUE TITLE
Missing coverage for ternary operator

### ISSUE THREAD

**Issue by kamilhark** [kwonoj/swc-plugin-coverage-instrument#234]
Hi

`const a = condition ? foo : bar`
the code for colleting the coverage is not injected into branches of ternary operator
it results in missing BRDA lines in lcov files (only DA lines are added)

for nyc it works fine, the branches are recognized.

Is it just missing or I can control it via configuration?

**Comment by kamilhark** [comment]
I investigated it a little bit, ternary operator does work but only if used outside a function, it's handled by `visit_mut_cond_expr`, it does not work inside a function body and then the `visit_mut_cond_expr` is never called.
unfortunately I don't know Rust enough to fix it.
reproduction:
```
---
name: ternary operator
code: |
  var x = args[0];
  var y = args[1];
  function foo() {
     return x > y ? x : y
  }
  output = foo();
tests:
  - name: covers then path
    args: [20, 10]
    out: 20
    lines: {'1': 1, '2': 1, '4': 1, '6': 1}
    branches: {'0': [1, 0]}
    statements: {'0': 1}
    functions: {'0': 1}
  - name: covers else path
    args: [10, 20]
    out: 20
    lines: { '1': 1, '2': 1, '4': 1, '6': 1 }
    branches: {'0': [0, 1]}
    statements: {'0': 1}
    functions: {'0': 1}
```
any chance you can take a look?

**Comment by kwonoj** [comment]
No, I'm occupied with daily work to make a PR at this moment. As said, I'm happy to review & accept PR.
